local-dev/src/components/GlobalStatus.js

```diff
@@ -48,15 +45,51 @@ class GlobalStatus extends Component {
             default:
                 break;
         }
+        return img;
+    }
+    setFavicon(status) {
+        // update favicon
+        var favicon = document.getElementById('favicon');
+        var img = this.statusToIcon(status);
 
         if (img) {
             favicon.href = img;
         }
     }
+    realDesktopNotify(message, icon) {
+        var options = {
+            body: 'Mozilla Engagement Engineering Status Board',
+            icon: icon
+        }
+        new Notification(message, options);
+    }
+    desktopNotify(message, icon) {
+        if (!('Notification' in window)) {
+            return;
+        }
+        // Let's check whether notification permissions have already been granted
+        else if (Notification.permission === 'granted') {
+            this.realDesktopNotify(message, icon);
+        }
+        // Otherwise, we need to ask the user for permission
+        else if (Notification.permission !== 'denied') {
+            Notification.requestPermission(function (permission) {
```

Comment:
Need to use an arrow function here to have the correct `this` below. (Or use `.bind`, but arrow functions are way cooler these days.)